### PR TITLE
remove 'Export-Package', 'Import-Package', 'Private-Package' for source bundle

### PR DESCRIPTION
**Pull request by missedone**
fix issue #32

### CODE REVIEW

File: src/main/groovy/org/standardout/gradle/plugin/platform/internal/util/bnd/BundleHelper.groovy [hunk 1]
```diff
@@ -67,8 +67,7 @@ class BundleHelper {
 				(Analyzer.BUNDLE_NAME): sourceArt.bundleName,
 				(Analyzer.BUNDLE_VERSION): sourceArt.modifiedVersion,
 				(Analyzer.BUNDLE_SYMBOLICNAME): sourceArt.symbolicName,
-//				(Analyzer.PRIVATE_PACKAGE): '*', // sources as private packages
-				(Analyzer.EXPORT_PACKAGE): '!', // no exports
+				(Analyzer.REMOVEHEADERS): '*-Package',	// no 'Export-Package', 'Import-Package', 'Private-Package' for source bundle
```
Comment (missedone) [inline src/main/groovy/org/standardout/gradle/plugin/platform/internal/util/bnd/BundleHelper.groovy]: see http://bnd.bndtools.org/instructions/removeheaders.html

**Comment by stempler** [comment]
Thanks a lot for the fix!

**Comment by missedone** [comment]
thanks @stempler , 
where can I found 1.6.0-SNAPSHOT `org.standardout:bnd-platform:1.6.0-SNAPSHOT`

**Comment by stempler** [comment]
You can find the SNAPSHOT build in this Maven repository:
https://oss.sonatype.org/content/repositories/snapshots

**Comment by missedone** [comment]
thx, the snapshot works

**Pull request merged**